pkg/operator/controller/ingress/controller.go

```diff
@@ -561,8 +561,29 @@ func setDefaultPublishingStrategy(ic *operatorv1.IngressController, platformStat
 					statusLB.ProviderParameters.GCP.ClientAccess = specClientAccess
 					changed = true
 				}
+			case operatorv1.IBMLoadBalancerProvider:
+				// The only provider parameter that is supported
+				// for IBM is the Protocol parameter.
+				var specProtocol, statusProtocol operatorv1.IngressControllerProtocol
+				if specLB.ProviderParameters != nil && specLB.ProviderParameters.IBM != nil {
+					specProtocol = specLB.ProviderParameters.IBM.Protocol
+				}
```

Comment:
Rather than checking `specLB.ProviderParameters != nil && specLB.ProviderParameters.IBM` here, please add a case in `setDefaultProviderParameters` to initialize these fields like we have for `operatorv1.GCPLoadBalancerProvider`.

Reply:
Moved specLB initialization to `setDefaultProviderParameters`.